### ISSUE TITLE
[Suggestion] Option to change the `currentColor`

### ISSUE THREAD

**Issue by RuiNtD** [vincerubinetti/svg-to-png#18]
Hello. It would be nice if there was an option to easily change the `currentColor` value, which is normally affected by the `color` CSS property.

**Comment by vincerubinetti** [comment]
This is a good idea, I often find myself manually replacing the `currentColor`s in my SVG source codes in the app. I guess this would just be another control next to the `background` input.

Explaining to the average user what `currentColor` is will be tricky. I think almost no one who uses this app will ever have an SVG with a `currentColor` in it, as it's probably mostly people who make SVGs in inkscape and google slides. You'd only ever have `currentColor` if you're writing SVGs by hand.

**Comment by RuiNtD** [comment]
It might be a good idea to only show the field if currentColor is actually being used and to just name the field "color". I dunno if it's sufficient to just detect this by checking for the string "currentColor". Another option to detect this could just be rendering the SVG twice with white and black as the colors and comparing the outputs. A small size should be sufficient for the test renders, like 50x50, since with the browser's anti-aliasing, even a small difference should result in a different render.

**Comment by vincerubinetti** [comment]
I think in almost all cases it would be sufficient to check if the source contains `currentColor` or `currentcolor`. Only case I can think of where that would fail is if the SVG was importing an external stylesheet. But I'm not sure this tool supports that anyway, because I'm not sure if canvas's `ctx.drawImage` supports it.

Regardless, I've come up with a more general solution. I'll be pushing an update in a bit and you'll see what I mean.

**Comment by vincerubinetti** [comment]
There's a PR for this here:

https://github.com/vincerubinetti/svg-to-png/pull/19

You'd have to run it locally to see how it works though. You'd need Node 22 and Bun, then run `bun install && bun dev`.

Basically I've added a new "Color" column that will "force" all colors to a specific color using an SVG filter. I've found myself wanting to do this many times. Or, if you put a `~` at the beginning of the color, it will just set `currentColor`.

I also added an unrelated "Trim" feature that I've needed many times.
